sdk/src/tx/baseTxSender.ts

```diff
@@ -210,6 +214,22 @@ export abstract class BaseTxSender implements TxSender {
 		}
 
 		if (response === null) {
+			if (this.confirmationStrategy === ConfirmationStrategy.Combo) {
+				try {
+					const rpcResponse = await this.connection.getSignatureStatus(
+						signature
+					);
+					if (rpcResponse?.value?.confirmationStatus) {
+						response = {
+							context: rpcResponse.context,
+							value: { err: rpcResponse.value.err },
```

Comment:
isn't this missing the value?

Reply:
no the Signature result is only an optional error

Reply:
https://solana-labs.github.io/solana-web3.js/types/SignatureResult.html